cloud_governance/policy/aws/nat_gateway_unused.py

```diff
@@ -8,10 +11,36 @@ class NatGatewayUnused(NonClusterZombiePolicy):
     to the user after 4 days and delete after 7 days.
     """
 
+    NAMESPACE = 'AWS/NATGateway'
+
     def __init__(self):
         super().__init__()
+        self._cloudwatch = CloudWatchOperations(region=self._region)
+
+    def __check_cloud_watch_logs(self, resource_id: str, days: int = 1):
+        """
+        This method returns weather the NatGateway is used in last input days
+        :param resource_id:
+        :param days:
+        :return:
+        """
+        end_time = datetime.datetime.utcnow()
+        start_time = end_time - datetime.timedelta(days=days)
+        response = self._cloudwatch.get_metric_data(start_time=start_time, end_time=end_time, resource_id=resource_id,
```

Comment:
Do you think unused 1 day is enough to decide to delete it ?

Reply:
no, here Iam passing the LastUsedDay Count starting from one. If I encounter the first day is unused, I'll increment the LastUsedDay.

Reply:
How much days we count till we can delete it ?